docs/source/algorithms/AlignDetectors-v1.rst

```diff
@@ -6,6 +6,13 @@
 
 .. properties::
 
+.. note:: As of 2021-01-04, this algorithm is officially deprecated.
```

Comment:
Thank you for adding detailed instructions on how to replace the deprecated algorithm